tests/test_garden.py

```diff
@@ -236,3 +237,126 @@ def test_register_metadata(garden_client, garden_required_fields_only, tmp_path)
     with open(tmp_path / "metadata.json", "r") as f:
         json_contents = f.read()
         assert json_contents == garden.json()
+
```

Comment:
Please split this up into smaller, focused python files. Any python file that starts with `test_` will be used by pytest

Reply:
done, there are now distinct `test_` files for `garden.py` and `models.py` - I've also moved all fixtures to a `conftest.py` (including the ones you'd written, happy to move those back to test_garden though if they should live in the tests file for whatever reason)